default_coverage_report.toml

```diff
@@ -0,0 +1,11 @@
+[tool.coverage.report]
+fail_under = 100
+show_missing = true
+omit = [
+  "tests/*",
```

Comment:
We probably shouldn't omit tests coverage

Reply:
Are you sure you want to do that, isn't coverage about implementation code. Adding test coverage could add false positives/negatives to the results and maybe even cause branch coverage to fail if the total scores fell below the threshold.